packages/rocketchat-slackbridge/settings.js

```diff
@@ -25,6 +25,15 @@ Meteor.startup(function() {
 			i18nDescription: 'Alias_Format_Description'
 		});
 
+		this.add('SlackBridge_FileUpload_Enabled', true, {
+			type: 'boolean',
+			enableQuery: {
+				_id: 'SlackBridge_Enabled',
+				value: true
+			},
+			i18nLabel: 'FileUpload'
```

Comment:
The term 'FileUpload' may be dubious. I suggest using a specific entry to mention uploading activated Slack Bridge files.